cachi2/core/package_managers/yarn/utils.py

```diff
@@ -27,3 +29,68 @@ def run_yarn_cmd(
     except subprocess.CalledProcessError as e:
         # the yarn command writes the errors to stdout
         raise PackageManagerError(f"Yarn command failed: {' '.join(cmd)}", stderr=e.stdout)
+
+
+VersionLike = Union[Version, str]
```

Comment:
nitpick: `packaging.Version` is different from `semver.Version`. We do use **a** `Version` object interchangeably throughout the code base IIRC so a type name of `SemVerLike` feels more appropriate for the casual code reader.

Even I missed the `semver` import for the first couple of minutes of this review and was puzzled on how would this close #660, until I found the import.

Reply:
Done.